flink-core/src/main/java/org/apache/flink/configuration/ClusterOptions.java

```diff
@@ -140,7 +140,7 @@ public class ClusterOptions {
     public static final ConfigOption<Boolean> ENABLE_FINE_GRAINED_RESOURCE_MANAGEMENT =
             ConfigOptions.key("cluster.fine-grained-resource-management.enabled")
                     .booleanType()
-                    .defaultValue(false)
+                    .defaultValue(true)
```

Comment:
In addition to changing the default value, I think we should annotate the configuration as `@Deprecated` and exclude it from documentation. Because we are expecting to remove the declarative slot manager in the next release, and this config will then become invalid.